c7n/resources/ec2.py

```diff
@@ -302,6 +302,56 @@ def __call__(self, i):
         return self.operator(map(self.match, volumes))
 
 
+@filters.register('termination-protected')
+class DisableApiTermination(Filter):
+    """EC2 instances with ``disableApiTermination`` attribute set
+
+    Filters EC2 instances with ``disableApiTermination`` attribute set to true.
+
+    :Example:
+
+    .. code-block:: yaml
+
+        policies:
+          - name: termination-protection-enabled
+            resource: ec2
+            filters:
+              - type: termination-protected
+
+    :Example:
+
+    .. code-block:: yaml
+
+        policies:
+          - name: termination-protection-NOT-enabled
+            resource: ec2
+            filters:
+              - not:
+                - type: termination-protected
+    """
+
+    schema = type_schema('termination-protected')
+    permissions = ['ec2:DescribeInstanceAttribute']
```

Comment:
so this should to be a tuple, else when your extending below this list gets larger on every use of the filter within a given cli execution, (aka its like a mutable default variable that gets bigger on every use). and then below you can create a new list from the tuple to ensure its just the values needed.

Reply:
Oops, sorry.